samples/bluetooth/hci_uart/overlay-all.conf

```diff
@@ -0,0 +1,69 @@
+CONFIG_BT_BUF_EVT_RX_COUNT=16
+
+CONFIG_BT_BUF_EVT_RX_SIZE=255
+CONFIG_BT_BUF_ACL_RX_SIZE=255
+CONFIG_BT_BUF_ACL_TX_SIZE=251
+CONFIG_BT_BUF_CMD_TX_SIZE=255
+
+CONFIG_BT_EXT_ADV=y
+CONFIG_BT_PER_ADV=y
+CONFIG_BT_ISO_BROADCASTER=y
+
+CONFIG_BT_PER_ADV_SYNC=y
+CONFIG_BT_PER_ADV_SYNC_MAX=2
+CONFIG_BT_ISO_SYNC_RECEIVER=y
+
+CONFIG_BT_ISO_MAX_CHAN=2
+
+CONFIG_BT_CTLR_ASSERT_HANDLER=y
+CONFIG_BT_CTLR_DTM_HCI=y
+
+CONFIG_BT_CTLR_RX_BUFFERS=9
+CONFIG_BT_CTLR_DATA_LENGTH_MAX=251
+
+CONFIG_BT_CTLR_PHY_CODED=y
+
+CONFIG_BT_CTLR_ADV_SET=3
+CONFIG_BT_CTLR_ADV_DATA_LEN_MAX=191
+CONFIG_BT_CTLR_SCAN_DATA_LEN_MAX=1650
+
+CONFIG_BT_CTLR_ADV_ISO=y
+CONFIG_BT_CTLR_ADV_ISO_PDU_LEN_MAX=251
+CONFIG_BT_CTLR_ISO_TX_BUFFERS=16
+CONFIG_BT_CTLR_ISO_TX_BUFFER_SIZE=251
+CONFIG_BT_CTLR_ADV_ISO_STREAM_MAX=2
+
+CONFIG_BT_CTLR_SYNC_ISO=y
+CONFIG_BT_CTLR_SYNC_ISO_PDU_LEN_MAX=251
+CONFIG_BT_CTLR_SYNC_ISO_STREAM_MAX=2
+CONFIG_BT_CTLR_ISOAL_SINKS=2
+
+CONFIG_BT_CTLR_ADVANCED_FEATURES=y
+CONFIG_BT_CTLR_ADV_AUX_SET=3
+CONFIG_BT_CTLR_ADV_AUX_PDU_BACK2BACK=y
+CONFIG_BT_CTLR_ADV_SYNC_SET=3
+CONFIG_BT_CTLR_ADV_SYNC_PDU_BACK2BACK=y
+CONFIG_BT_CTLR_ADV_DATA_BUF_MAX=6
+
+# Increase the below to receive interleaved advertising chains
+# CONFIG_BT_CTLR_SCAN_AUX_SET=2
```

Comment:
Why is this commented out?

Reply:
Removed comment and set it to default value.